Analysis/Tasks/PWGMM/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+# Copyright CERN and copyright holders of ALICE O2. This software is distributed
+# under the terms of the GNU General Public License v3 (GPL Version 3), copied
+# verbatim in the file "COPYING".
+#
+# See http://alice-o2.web.cern.ch/license for full licensing information.
+#
+# In applying this license CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization or
+# submit itself to any jurisdiction.
+
+o2_add_dpl_workflow(dNdetaRun2Tracklets-analysis
+                    SOURCES dNdetaRun2Tracklets.cxx
+                    PUBLIC_LINK_LIBRARIES O2::Framework O2::AnalysisDataModel O2::AnalysisCore O2::DetectorsBase
```

Comment:
Probably you do not need DetectorBase library, most likely AnalysisCore as well

Reply:
Removed DetectorBase library, AnalysisCore is necessary for evsel